### ISSUE TITLE
Fusionpbx does git clone rather than using FreeBSD port

### ISSUE THREAD

**Issue by tonyhain** [fusionpbx/fusionpbx#2535]
I had installed the FS port on FreeBSD 11 with all modules, but Fusionpbx did a git clone and overwrote that install. The only reason I noticed was that mod_blacklist was failing to load with the error:
2017-04-23 22:18:03.792848 [CRIT] switch_loadable_module.c:1522 Error Loading module /usr/local/freeswitch/mod/mod_blacklist.so
**Cannot open "/usr/local/freeswitch/mod/mod_blacklist.so"**
--- after a symlink from /usr/local/lib/freeswitch... to /usr/local/freeswitch... it worked
2017-04-23 22:22:37.833061 [INFO] mod_blacklist.c:104 Loaded list [FCC]

There are a bunch of different-size-with-the-same-name .so files between the .../mod/ locations. Until I start over I can't tell which set are from the port vs. the git clone. In any case this situation is wrong. Fusionpbx should be using /usr/ports/net/freeswitch to avoid any weirdness, and should have recognized that FS was already installed.

**Comment by markjcrane** [comment]
FusionPBX doesn't compile mod_blacklist by default because we don't use it by default.

As far as saying its wrong then the FreeSWITCH port is a couple versions behind which I think is also debateably wrong. So ability to compile or use the port is a good thing. Please keep in mind the script for FreeBSD is new and using the port is on the to do list.

**Comment by markjcrane** [comment]
This issue was better for the fusionpbx-install.sh repo since its to do with the install.

**Comment by markjcrane** [comment]
I'll close the issue when I have the port install working as part of the install script.

**Comment by tonyhain** [comment]
I agree the option to pull master is good to keep. I was just surprised that it did that when FS was already installed. In either case, there needs to be consistency between the sources as to where lib files end up.

**Comment by markjcrane** [comment]
I worked on the FreeSWITCH package for FreeBSD for more than 10 hours today and after getting almost everything working I hit a major snag no PostgreSQL driver included in the FreeSWITCH package. I want to use the pre-compile package and so port will have to be fixed.

The FreeSWITCH port Makefile is missing the --enable-core-pgsql-support and maybe add Postgresql Client as a port dependency.

CONFIGURE_ARGS= --with-pkgconfigdir=${PREFIX}/libdata/pkgconfig \
                --enable-zrtp \
                --enable-core-pgsql-support

The challenge is likely getting the port maintainer to add this.

**Comment by markjcrane** [comment]
Waiting on the FreeBSD FreeSWITCH port developers to improve the  port and include the PostgreSQL drivers. They are preventing this from working https://www.freshports.org/net/freeswitch/ 

Please contact them to see if they will add --enable-core-pgsql-support to the CONFIGURE_ARGS.
The only other way is to manually configure odbc to use PostgreSQL if they have ODBC enabled.
